hydroshare/settings.py

```diff
@@ -239,6 +239,10 @@
 THUMBNAIL_PRESERVE_FORMAT = True
 THUMBNAIL_QUALITY = 95
 
+# Allow PIL to ignore imgs with lots of metadata
```

Comment:
Scott, this fixes the issue we had when attempting to upload [Dan Ames' photo](https://www.hydroshare.org/user/30/)

Reply:
lol, very good find!

Reply:
Do you still need to do the im.exists?  Are there other profile images that thumbnails silently fail to be created for?

Reply:
Good call thanks for checking. There are some other profile images failing (some listed in #4676) but I'm convinced they have been missing for a while and are not related to sorl-thumbnail.

I haven't noticed any other failing profile images.
This issue https://github.com/jazzband/sorl-thumbnail/issues/442 made me wonder how it would handle non-ascii filenames. But I just tested a few and they worked fine

Reply:
>made me wonder how it would handle non-ascii filenames.
I think most (all?) of our image names are turned into uids at upload.

Sounds like the potential slow down from `im.exists` isn't an issue or is negligible.

Reply:
Looks like we [append a hash the profile picture names](https://github.com/hydroshare/hydroshare/blob/develop/theme/utils.py#L6). But it is _appended_ not fully overwritten so non-ascii chars make it all the way into our static files.

[Group images aren't hashed at all](https://github.com/hydroshare/hydroshare/blob/1.55.2/hs_core/views/__init__.py#L1279)

Reply:
@devincowan Group images and community images are hashed as well to ensure uniqueness of image names, as shown [here](https://github.com/hydroshare/hydroshare/blob/develop/theme/utils.py#L20-L29)

Reply:
@devincowan That is right. The hash code just append a hash to make sure the image file names are unique. I guess some check for non-ascii chars to perhaps remove them from the filename could be added if non-ascii chars caused profile image missing issues.